server/chat-plugins/uno.ts

```diff
@@ -526,33 +490,28 @@ class UnoGame extends Rooms.RoomGame {
 		this.nextTurn();
 	}
 
-	/**
-	 * @param {UnoGamePlayer} player
-	 * @param {number} count
-	 * @return {Card[]}
-	 */
-	onDrawCard(player, count) {
+	onDrawCard(player: UnoGamePlayer, count: number) {
 		if (typeof count === 'string') count = parseInt(count);
 		if (!count || isNaN(count) || count < 1) count = 1;
-		let drawnCards = /** @type {Card[]} */ (this.drawCard(count));
+		const drawnCards: Card[] = this.drawCard(count);
```

Comment:
```suggestion
		const drawnCards = this.drawCard(count);
```

I think this is inferrable too.